langlib/lang.string/src/main/ballerina/string.bal

```diff
@@ -47,14 +59,26 @@ public isolated function iterator(string str) returns object {
 
 # Concatenates zero or more strings.
 #
+# ```ballerina
+# string hostname = "http://worldtimeapi.org";
+# string continent = "Asia";
+# string city = "Colombo";
+# string worldTimeAPIUrl = hostname.concat("/api/timezone/", continent, "/", city);
```

Comment:
Alternatively we can use `string:concat(hostname, "/api/timezone", continent, "/", city)`. Should we add that as well?

Reply:
Similar discussion: https://github.com/ballerina-platform/ballerina-lang/pull/38905#discussion_r1037781532.

Reply:
Incorporated the alternative approach as another example here. I felt approach suggested by Imesha is a common way